pkg/scheduler/scheduler.go

```diff
@@ -303,16 +305,19 @@ func (s *Scheduler) getPlacement(resourceBinding *workv1alpha2.ResourceBinding)
 	return placement, string(placementBytes), nil
 }
 
-func (s *Scheduler) getClusterPlacement(crb *workv1alpha2.ClusterResourceBinding) (policyv1alpha1.Placement, string, error) {
-	var placement policyv1alpha1.Placement
+func (s *Scheduler) getClusterPlacement(crb *workv1alpha2.ClusterResourceBinding) (*policyv1alpha1.Placement, string, error) {
+	var placement *policyv1alpha1.Placement
 	policyName := util.GetLabelValue(crb.Labels, policyv1alpha1.ClusterPropagationPolicyLabel)
+	if policyName == "" {
```

Comment:
In fact, this judgment is unnecessary. For those resourceBindings without labels, the scheduler will filter them out.

Reply:
Thanks for your reminder.